modules/ROOT/pages/organization.adoc

```diff
@@ -19,47 +19,63 @@ An Anypoint Platform user who has been granted organization administrator privil
 [NOTE]
 Business groups are not enabled by default in a new Anypoint Platform account. To activate business groups in your organization, contact your MuleSoft representative.
 
+== Access the Organization Administration
+
+. Sign into your Anypoint Platform account as a user with the Organization Administrator role.
+. Select the *Access Management* option from the main page, or select it from the left side bar by clicking the _menu_ button in the top left corner. +
+The *Organization* page is displayed, where you can view organizations and business groups and add new business groups (if enabled).
+
 == The Organization Owner
 
-The user who first signs up for the Anypoint Platform organization is known as the *Organization Owner*. This is not a role but an identifier for this single user, who inherits the Organization Administrator role by default.
+The user who first signs up for an Anypoint Platform account is designated as the organization owner. This is not a role but an identifier for this single user, who is assigned the Organization Administrator role by default. Other users can be assigned the Organization Administrator role, but there is only one organization owner per Anypoint Platform account (master organization). The owner can transfer ownership of the account to another user, but there cannot be more than one owner at once.
 
-When the Organization Owner creates a business-group, an owner must be assigned to it. This assigned user holds an Administrator role within that business group by default.
+When the organization owner creates a business group, an owner must be assigned to it. This assigned user holds an Organization Administrator role within that business group by default.
 
-== Organization Settings
+Because the organization owner always has the Organization Administrators role in the master organization, that user always has that same role in all business groups within the organization, so this user is visible in all business groups. The Organization Administrators role cannot be removed from the organization owner in the master organization or any business group.
```

Comment:
while this is true, it doesn't explain
1. why this is so (because "Users that have been assigned the role of "Organization Administrators" in the Org or any BG automatically are assigned that same role in any child BG created below that Org/BG. They thereby immediately become visible in that child BG.")
2. the limitations of this statement ("This does not apply to users that are assigned the "Organization Administrators" role _after_ a child BG has been created.")
If these 2 aspects are explained somewhere else then maybe you can link to that description from here?

Reply:
Hi Gerald,
Both of those points are made in the section "Creating Roles and Handling Membership to Business Groups"